tests/test.payment-app.jsx

```diff
@@ -68,6 +68,7 @@ describe('Payment App', function() {
 
   it('should render a purchase page', function() {
     var user = {
```

Comment:
I think it's time to should start doing this instead:
````
import { defaultUser } from 'actions/user';
...
var user = Object.assign({}, defaultUser, {email: 'f@f.com', signedIn: true});
````

There are a couple tests now that can benefit from that.